azure-pipelines.yml

```diff
@@ -90,10 +90,23 @@ jobs:
 
 - job: macOS
   pool:
-    vmImage: macOS-10.13
+    vmImage: macOS-latest
   steps:
 
-  - bash: ./build.sh --target=Test --configuration=Release
+  - task: UseDotNet@2
+    displayName: 'Install .NET Core SDK'
+    inputs:
+      version: 3.1.x
+      
+  - task: UseDotNet@2
+    displayName: 'Install .NET Core runtime 2.1'
+    inputs:
+      packageType: runtime
+      version: 2.1.x
+
+  - bash: |
+     dotnet tool install --global Cake.Tool
+     dotnet cake --target=Test --configuration=Release
```

Comment:
I haven't seen this before. Cute! 🙂

Reply:
I could have gotten some inspiration from https://github.com/nunit/nunit/blob/master/azure-pipelines.yml#L84-L87 😄 (I had to do this otherwise cake could not use the installed SDK),